rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -418,8 +418,7 @@ if [ ${CHECK_BACKSLASH} -ne 1 ]; then
   echo -n "INFO: Rudder database is not backslash compliant, fixing..."
   ${PSQL} -t -d rudder -c "alter database rudder set standard_conforming_strings=true;" >/dev/null 2>&1
   echo " Done"
-  echo -n "INFO: PostgreSQL and Rudder will be restarted..."
-  service postgresql restart >/dev/null 2>&1
```

Comment:
No. This change does not require a restart of PostgreSQL. (cf. http://www.depesz.com/2010/07/21/waiting-for-9-1-standard_conforming_strings-on/)